# Paraguay - Developers Survey year 2021

In [3]:
# Importing Libs
import pandas as pd
import matplotlib.pyplot as plt
%matplotlib inline

In [4]:
plt.rcParams['figure.figsize'] = [10, 5]

In [6]:
df = pd.read_csv('enc_devs_py.csv')

### First lets see what columns we have.

In [161]:
for i in df.columns:
    print(i)

Marca temporal
Modalidad
Trabajo
Tipo de trabajo
DevOps
Codificar por trabajo
Experiencia en TI
Experiencia en codificación 
Primera línea de código en una máquina
Educación
Carreras o especialidades
Importancia educación formal
Género
Edad actual 
Salario mensual
Programación, scripting y lenguajes de marcado
Librerías o web frameworks
Otras frameworks, librerías y herramientas
Base de datos
Plataformas
Lenguaje favorito
Lenguaje más odiado
Framework favorito
Framework más odiado
Herramientas


In [162]:
df.shape

(342, 25)

- Contamos con un df de 342 filas y 25 columnas.

# DATA DICTIONARY

### Modalidad : Tipo de trabajo que realiza. Presencial / Remoto / Hibrido.
### Trabajo: Nacional o extranjeros.
### Tipo de trabajo: Rol del trabajo que realiza. Dev/Designer
### DevOps: Rol del trabajo en DEVOPS.
### Codificar por trabajo: Su trabajo requiere programacion?
### Experiencia en TI: Años trabajando en TI.
### Experiencia en codificación: Años de experiencia en codificacion.
### Primera línea de código en una máquina: Edad en la que realizo la primera linea de codigo.
### Educación: Tipo de educacion recibida.
### Carreras o especialidades: Carrera o especializacion.
### Importancia educación formal: Considera que la educacion formal es importante?
### Género : Sexo de la persona que responde.
### Edad actual : Edad actual al momento de responder la pregunta.
### Salario mensual : Salario mensual persivido.
### Programación, scripting y lenguajes de marcado: Lenguajes de programacion usados.
### Librerías o web frameworks: Librerias uilizadas.
### Otras frameworks, librerías y herramientas: Otras librerias utilizadas.
### Base de datos: Bases de datos utilizadas.
### Plataformas: Plataformas de trabajo.
### Lenguaje favorito: Lenguaje favorito.
### Lenguaje más odiado: Lenguaje mas odiado.
### Framework favorito: Framework favorito.
### Framework más odiado: Framework mas odiado.
### Herramientas: Herramientas utilizadas para el desarrollo de sus actividades.